dev-python/matplotlib/matplotlib-3.3.2.recipe

```diff
@@ -0,0 +1,89 @@
+SUMMARY="Pure python plotting library with matlab like syntax"
+DESCRIPTION="Matplotlib is a plotting library for the Python programming \
+language and its numerical mathematics extension NumPy. It provides an \
+object-oriented API for embedding plots into applications using \
+general-purpose  GUI toolkits like Tkinter, wxPython, Qt, or GTK+. There is \
+also a procedural \"pylab\" interface, designed to closely resemble that of \
+MATLAB, though its use is discouraged."
+HOMEPAGE="https://matplotlib.org/"
+COPYRIGHT="2002 - 2012 John Hunter, Darren Dale, Eric Firing, Michael \
+Droettboom and the Matplotlib development team \
+2012 - 2020 The Matplotlib development team"
+LICENSE="Python" # "PSF-based" actually
```

Comment:
You need to add the license here too (as a seperate file), see for instance:
https://github.com/haikuports/haikuports/blob/c2a214e6473117adfdbf84d0f792341b30e1bbd7/dev-python/lxml/licenses/PSF-2
https://github.com/haikuports/haikuports/blob/c2a214e6473117adfdbf84d0f792341b30e1bbd7/dev-lang/python/licenses/Python

Reply:
Got it